.github/workflows/build.yaml

```diff
@@ -45,3 +45,14 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  docs-build:
+    if: ${{ startsWith(github.ref, 'refs/heads/branch-') }}
+    needs: cpp-build
```

Comment:
```suggestion
```

The `build_docs.sh` script doesn't depend on any artifacts from the `cpp-build` stage, therefore this dependency can be removed.